import_and_save.py

```diff
@@ -0,0 +1,85 @@
+import click
+import csv
+from importlib import import_module
+
+def save_into_mysql(filename, username, password):
```

Comment:
Too much logic in single function, break in to smaller functions and re-use them as necessary.